.github/workflows/stop-environment.yml

```diff
@@ -0,0 +1,66 @@
+name: Stop AWS Environment
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "The environment you wish to start"

```

Comment:
@phenom5886 , I think there's a copy-paste mistake here and that this should be "stop" instead of "start".